accepted/2018/automatic-redirects-by-default/automatic-redirects-by-default.md

```diff
@@ -1,7 +1,7 @@
 # Automatic Binding Redirection By Default
 
-**PM** [Immo Landwerth](https://github.com/terrajobst) |
-**Dev** [Rainer Sigwald](https://github.com/rainersigwald) |
+**Owner** [Immo Landwerth](https://github.com/terrajobst) |
+**Owner** [Rainer Sigwald](https://github.com/rainersigwald) |
```

Comment:
If we only have a single role, I would prefix the list with a single **Owner** (regardless how many there are)